<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QWidget" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarms and Clock</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_timer">
      <attribute name="title">
       <string>Timer</string>
      </attribute>
      <widget class="QSpinBox" name="timer_h">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="timer_m">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="timer_s">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_timer_start">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>210</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px; border-radius: 30px; background-color: #000; color: #fff;</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_timer_pause">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px; border-radius: 30px; background-color: #000; color: #fff;</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_timer_reset">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>210</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px; border-radius: 30px; background-color: #000; color: #fff;</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Alarm</string>
      </attribute>
      <widget class="QSpinBox" name="alarm_m">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="alarm_h">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="alarm_s">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>101</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="alarm_checkbox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>220</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>16</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>80</y>
         <width>16</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>24-H Time Format</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Stopwatch</string>
      </attribute>
      <widget class="QPushButton" name="btn_stopwatch_start">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px; border-radius: 30px; background-color: #000; color: #fff;</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_pause">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>190</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px; border-radius: 30px; background-color: #000; color: #fff;</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stopwatch_reset">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>190</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px; border-radius: 30px; background-color: #000; color: #fff;</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_stopwatch">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Seven Segment</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
